docs/source/json-api/index.rst

```diff
@@ -1302,3 +1302,27 @@ Example:
     ]
 
 The output stream has the same format as the output from the `Contracts Query Stream`_. We further guarantee that for every ``archived`` event appearing on the stream there has been a matching ``created`` event earlier in the stream.
+
+You may supply an optional ``offset`` for the stream, exactly as with
+query streams.  However, you must supply with each ``{templateId,
+key}`` pair an ``offsetHint``, which is the contract ID currently
+associated with that pair at the point of the given offset, or
+``null`` if no contract ID was associated with the pair at that
+offset.  For example, with the above keys, if you had one ``"abc123"``
+contract but no `"def345"` contract, you might specify:
+
+.. code-block:: json
+
+    [
+        {"templateId": "Account:Account", "key": {"_1": "Alice", "_2": "abc123"},
+         "offsetHint": "#1:0"},
+        {"templateId": "Account:Account", "key": {"_1": "Alice", "_2": "def345"},
+         "offsetHint": null}
+    ]
```

Comment:
@hurryabit we should rename `offsetHint` to `contractIdHint` or `contractIdOffsetHint`.

Remember we have offset ticks defined as:
```
{"events":[],"offset":"5609"}
```
The `offset` value from the tick and `offsetHint` are two different things.

Reply:
let's merge it... and discuss the field name during the catch-up meeting.

Reply:
Yes, I think `contractIdHint` (or maybe `contractIdAtOffset`?) would be a better name.

Reply:
> `contractIdAtOffset`

What we've decided on.